src/fork_gc.c

```diff
@@ -752,15 +761,31 @@ static int ForkGc_PeriodicCallback(RedisModuleCtx *ctx, void *privdata) {
   int ret_val = 1;
 
   size_t totalCollectedBefore = gc->stats.totalCollected;
+  pid_t ppid_before_fork = getpid();
```

Comment:
Just `parent_pid`

Reply:
no, its parent id before fork (it might change after fork if the parent crashed)